### PR TITLE
Fixes Careers Page

### PR DESCRIPTION
**Pull request by Adebesin-Cell**
Fixes Careers Page

links:
https://iq-wiki-git-fixes-careers-page-prediqt.vercel.app/careers

Screenshots:
localhost_4000_careers

fixes https://github.com/EveripediaNetwork/issues/issues/1024

### CODE REVIEW

File: src/data/CareersData.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  link: string
+  location: string
+}
+
+export const AllCareers: CareersDatatype[] = [
+  {
+    id: 1,
+    title: 'AI developer',
+    description:
+      'Are you an AI Developer with an insatiable appetite for a challenge? Then this job opportunity is for you! We are looking for an ambitious and creative AI Developer to join our team and help solve some of the most complex problems in the tech industry. You will have the opportunity to work with the most cutting edge technologies and develop innovative solutions that have the potential to make a real impact on the world. If you’re looking to take your AI development career to the next level, then this is the job for you! So, c’mon – let’s get coding and make some magic!',
```
Comment (kesar) [inline src/data/CareersData.ts:14]: make sure it support line jumps

File: src/data/CareersData.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  id: number
+  title: string
+  description: string
+  link: string
+  location: string
+}
+
+export const AllCareers: CareersDatatype[] = [
+  {
+    id: 1,
```
Comment (kesar) [inline src/data/CareersData.ts:11]: i dont think we need ids, we can use key in the react loop

Comment (Adebesin-Cell) [reply src/data/CareersData.ts:11]: Noted @kesar. I'll remove it.

File: src/data/CareersData.ts (hunk 1 above)
Comment (Adebesin-Cell) [reply src/data/CareersData.ts:14]: I'll investigate that @kesar.

**Comment by kesar** [comment]
do we need a 1900x900 images for the hero? and dont we have those heros somewhere already?

also yarn.lock is updated without any new package added, so this is most likely something coming from other previous PR. probably worth revert yarn

**Comment by Adebesin-Cell** [comment]
> do we need a 1900x900 images for the hero? and dont we have those heros somewhere already?
> 
> we have similar / same brainies bg (that i really love :P) in homepage and as header in profile page if user didnt setup one. I think we should try to reuse them if possible
> 
> also yarn.lock is updated without any new package added, so this is most likely something coming from other previous PR. probably worth revert yarn

Okay @kesar, I'll add the fix now. I'll also revert yarn.

**Comment by Adebesin-Cell** [comment]
> code looks great now. kudos! the only missing things are linting & line jumping in job texts 👍🏻

Thanks @kesar. I'll look into how to implement line jumps and fix the lints.

**Comment by Softdev1** [comment]
Well done @Adebesin-Cell  looks great

**Review by Simplypheyie96 — CHANGES_REQUESTED** [review]
- The card in dark mode should be in gray700
image

- we're hiring tag is missing in footer
image

Brainies background not visible in dark mode

**Review by Simplypheyie96 — APPROVED** [review]

**Review by Royal-lobster — APPROVED** [review]

**Pull request merged**